src/demo/ui.jsx

```diff
@@ -260,28 +262,18 @@ let Button = class Button extends React.Component {
 Button = Radium(Button);
 
 const instructionsText = {
-  intro: [
-    {
-      heading: 'Introduction',
-      text: [
-        'In the following activity we’ll learn about artificial intelligence (AI) and machine learning.',
-        'With machine learning we use data to train the computer to recognize patterns.',
-        'Watch the video to learn more!'
-      ]
-    }
-  ],
-
   fishvtrash: [
     {
-      heading: 'Train A.I. to Clean Ocean',
+      heading: 'Train AI to Clean the Ocean',
       text: [
+        'In the following activity we’ll learn about <b>artificial intelligence</b> (AI) and <b>machine learning</b>.',
```

Comment:
Will this work on studio.code.org?  Due to our webfont, I think we might need to refer to a specific font-face...

Reply:
ohh, you're right. i'll revert that change here and pull it out into its own PR so the rest of this can be merged